root/static/scripts/common/entity.js

```diff
@@ -30,6 +38,12 @@ const formatTrackLength = require('./utility/formatTrackLength');
                 toJSON(result, this[key], key);
             }
             return result;
+        },
+
+        renderArtistCredit: function (ac) {
+            return ReactDOMServer.renderToStaticMarkup(
+                <ArtistCreditLink artistCredit={ko.unwrap(ac)} />
```

Comment:
The knockout stuff used to call https://github.com/metabrainz/musicbrainz-server/pull/263/files#diff-37e659b91af89773d96c6b5faa90d18cL462 to generate an html string for artist credits. Now it calls this, which generates static HTML from the react component we have. So one less instance of our custom knockout rendering crap.